Lombiq.Npm.Targets.targets

```diff
@@ -63,13 +63,105 @@
            AlwaysCreate="true" />
   </Target>
 
+  <UsingTask TaskName="CopyVendors"
+             TaskFactory="RoslynCodeTaskFactory"
+             AssemblyFile="$(MSBuildToolsPath)\Microsoft.Build.Tasks.Core.dll" >
+    <ParameterGroup>
+      <ProjectPath ParameterType="System.String" Required="true" />
+      <ProjectFilePath ParameterType="System.String" Required="true" />
+      <LibraryPath ParameterType="System.String" Required="true" />
+    </ParameterGroup>
+    <Task>
+      <Using Namespace="System" />
+      <Using Namespace="System.IO" />
+      <Using Namespace="System.Linq" />
+      <Using Namespace="System.Xml.Linq" />
+      <Using Namespace="System.Xml.XPath" />
+      <Code Type="Fragment" Language="cs">
```

Comment:
This is a bit too much C# in XML for my taste. `ExecWithMutex` in NE shows an example of how to implement such tasks in C# code files, what would be better.

Reply:
Thanks, that sounds great! I will look into it.